fidesctl/src/fidesctl/cli/util_comands.py

```diff
@@ -0,0 +1,131 @@
+"""Contains all of the Utility-type CLI commands for Fidesctl."""
+import os
+
+import click
+import requests
+import toml
+
+from fidesctl.cli.options import yes_flag
+from fidesctl.cli.utils import (
+    handle_cli_response,
+    pretty_echo,
+)
+from fidesctl.core import api as _api
+from fidesctl.core.utils import echo_green, echo_red
+
+
+@click.command()
+@click.pass_context
+def init(ctx: click.Context) -> None:
```

Comment:
Should this take an optional argument to specify where to place the `.fides` dir? It can fall back to `.`, but I think it could be potentially unclear otherwise.

Reply:
yeah I can do that. I was imagining that people should probably be running `fidesctl init` from the directory they want to create it in, but its trivial for us to add that feature so we might as well!